pocket-dark.user.css

```diff
@@ -185,7 +186,7 @@
   .coreform-container .btn-secondary, .coreform-container .btn-secondary:hover {
     color: var(--white);
   }
-  .coreform-container p > a {
+  .coreform-container p>a {
```

Comment:
```suggestion
  .coreform-container p > a {
```

Reply:
@the-j0k3r I'm assuming this should also apply to L325 and L367?

Reply:
dont bother here too much, when this is merged,I'll run perfectionist `npm run clean` and these should all be picked up and corrected. though Im wondering about the spaces after colons why they werent fixed by it.

Reply:
Yeah the `npm run clean` missed these when I ran it earlier.